mail_track_diff_only/tests/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# flake8: noqa
```

Comment:
why you are disabling flake8 here?

Reply:
Copyright updated and wrong disable of flake8 removed